XSIAM_PB_User_added_to_local_administrator_group (#36612)
demisto/content: Packs/Core/Playbooks/playbook-User_added_to_local_administrator_group_using_a_PowerShell_command_README.md

This playbook is designed to handle the alert
'User added to local administrator group using a PowerShell command'

The playbook executes the following stages:

Investigation:
Check the following parameters to determine if remediation actions are needed:
- XDR alerts related to the hostname by MITRE tactics indicating malicious activity.

> - Whether the process is signed.

Reviewer: ```suggestion
- Whether the process is unsigned.
```

Remediation:

> Handles malicious alerts by terminating the relevant processes and removing the user from the local Administrators group.

Reviewer: ```suggestion
Handles malicious alerts by terminating the relevant processes and requesting the analyst's approval to remove the user from the local Administrators group.
```

Handles non-malicious alerts identified during the investigation.

## Dependencies

This playbook uses the following sub-playbooks, integrations, and scripts.

### Sub-playbooks

This playbook does not use any sub-playbooks.

### Integrations

> * CortexCoreIR

Reviewer: ```suggestion
CortexCoreIR
```
